doc/tests/mapping.rst

```diff
@@ -0,0 +1,13 @@
+``mapping``
+============
```

Comment:
```suggestion
===========
```

Reply:
Done in https://github.com/twigphp/Twig/pull/3859/commits/9ecaf164fcf9de0a2f782153dbfa235d88a4efb6